pages/realms/components/RealmsDashboard.tsx

```diff
@@ -93,3 +105,9 @@ export default function RealmsDashboard({
     </div>
   )
 }
+
+const Badge: React.FC<{ count: number }> = ({ count }) => (
```

Comment:
nit: Can this be just a functional component? Do we need the explicit typing?

Reply:
sure :) which style do you prefer? I'll be consistent with whichever one you say

```tsx
const ConstWithReactFunctionSignature: React.FC<{ count: number }> = ({ count }) => <span>{count}</span>

const ConstWithType = ({ count }: { count: number }) => <span>{count}</span>
// ^ my slight preference, but very happy to go with any

const ConstWithoutType = ({ count }) => <span>{count}</span>

function FunctionWithType({ count }: { count: number }) {
  return <span>{count}</span>
}

function FunctionWithoutType({ count }) {
  return <span>{count}</span>
}
```

Reply:
I prefer your preference too :) Can we enforce it through a linter?
I would like to enable strict types checks at some point